php-frontend/src/main/java/org/sonar/php/tree/symbols/SymbolVisitor.java

```diff
@@ -165,11 +165,20 @@ public void visitClassDeclaration(ClassDeclarationTree tree) {
 
   @Override
   public void visitAnonymousClass(AnonymousClassTree tree) {
+    // the arguments are passed from the outer scope
+    scan(tree.arguments());
 
     enterScope(tree);
     classScope = currentScope;
     createMemberSymbols(tree);
-    super.visitAnonymousClass(tree);
+
+    // we've already scanned the arguments
+    scan(tree.members());
+    scan(tree.superInterfaces());
+    if (tree.superClass() != null) {
+      scan(tree.superClass());
+    }
```

Comment:
I would prefer to keep the same order as in [AnonymousClassTreeImpl.childrenIterator](https://github.com/SonarSource/sonar-php/blob/master/php-frontend/src/main/java/org/sonar/php/tree/impl/expression/AnonymousClassTreeImpl.java#L156): 
* `superClass`
* `superInterfaces`
* `members`